src/main/java/org/example/crmsystem/configuration/SessionConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package org.example.crmsystem.configuration;
+
+import org.apache.logging.log4j.plugins.Singleton;
+import org.example.crmsystem.entity.*;
+import org.hibernate.SessionFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@ComponentScan(basePackages = "org.example.crmsystem")
+@PropertySource("classpath:application.properties")
+public class SessionConfiguration {
+    @Bean
+    @Singleton
+    public SessionFactory sessionFactory() {
+        org.hibernate.cfg.Configuration configuration = new org.hibernate.cfg.Configuration();
+
```

Comment:
I recommend that you use property files for these types of configurations. 
This means using the @Value annotation to provide the data. 

Real-life example: In a real project, there are multiple environments including UAT, QA for testing, and Production, and they all use different database instances, hard coding of database credentials is not allowed.